app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">My Meetings</string>
+    <string name="app_name">My Events</string>
```

Comment:
I was thinking we can name the app as `Calendar Events Reminder`, what do you think?

Reply:
I think "Calendar Events Reminder" is too lengthy.
App name might not visible in launcher.
We have to come up with a nice name based on App Store Optimization.